src/design-pages/passkey_authentication.rst

```diff
@@ -0,0 +1,313 @@
+Passkey - Authentication in a centralized environment
+=======================================================
+
+Related ticket(s):
+------------------
+ * https://github.com/SSSD/sssd/issues/6228
+
+Problem statement
+-----------------
+The use of new tools to authenticate users, such as 2FA, U2F and FIDO2, is
+becoming increasingly popular and currently SSSD doesn’t provide any way to use
+the latter. This diminishes the value provided by SSSD in some environments
+like big companies and governments, where the usage of these authentication
+mechanisms is becoming a common pattern.
+
+SSSD should provide a way to authenticate users that use a passkey. This would
+add more value by attracting the types of users mentioned in the previous
+paragraph.
+
+For the purpose of this feature, passkey is a FIDO2 compatible device supported
+by the libfido2 library.
+
+Use cases
+---------
+As a user, I want to walk to any system on a network and log in (desktop and/or
+terminal) using a passkey device connected to the system. The user credentials
+will be stored in a central management system like FreeIPA or Active Directory.
+
+Solution overview
+------------------------
+The objective is to use a passkey to locally authenticate a user against 
+centralized identity management system. For that purpose an integration with an
+LDAP server like AD or FreeIPA is needed. SSSD should get the user credentials
```

Comment:
s/SSSD/SSSD clients/

Also, I would explicitly state that first the passkey data needs to be added to the LDAP directory server.

Reply:
I did both things

Reply:
Thanks, LGTM.